manimlib/mobject/svg/text_mobject.py

```diff
@@ -46,7 +47,17 @@ class Text(SVGMobject):
     def __init__(self, text, **config):
         self.full2short(config)
         digest_config(self, config)
-        self.lsh = self.size if self.lsh == -1 else self.lsh
+        if self.size:
+            warnings.warn(
+                "self.size has been deprecated and will "
+                "be removed in future.",
+                DeprecationWarning
+            )
+            self.font_size = self.size
+        if self.lsh == -1:
+            self.lsh = self.font_size + self.font_size * 0.3
```

Comment:
This 0.3 should probably be a constant or parameter defined elsewhere.

Reply:
Done https://github.com/3b1b/manim/pull/1553/commits/5765ab9055598b756c448e473ad7c594e1cb406a